lib/main_micro_alp.pm

```diff
@@ -45,15 +45,23 @@ sub load_config_tests {
 }
 
 sub load_boot_from_disk_tests {
-    loadtest 'installation/bootloader_start' if is_s390x();
+    # add additional image handling module for svirt workers
+    if (is_s390x()) {
+        loadtest 'installation/bootloader_start';
+    } elsif (is_vmware()) {
+        loadtest 'installation/bootloader_svirt';
+        loadtest 'installation/bootloader_uefi';
+    }
+
     if (check_var('FIRST_BOOT_CONFIG', 'wizard')) {
         loadtest 'jeos/firstrun';
+    } elsif (check_var('FIRST_BOOT_CONFIG', 'cloud-init')) {
+        loadtest 'boot/cloud_init';
     } elsif (is_s390x()) {
         loadtest 'boot/boot_to_desktop';
     } else {
         loadtest 'microos/disk_boot';
     }
-    loadtest 'boot/cloud_init' if (check_var('FIRST_BOOT_CONFIG', 'cloud-init'));
 
```

Comment:
Let we pay attention here if what we want 'cause:
moving this last indipendent `if cloud-init` into `switch` above, at its occurrence the next eventual `s390x` case will be skipped; 
in the old version,instead, both the 2 option could happen and run.

Reply:
Idk what exactly `boot/cloud_init` does, I guess some post cloud-init checks but as long as it at least logs the system which I would expect then it is doing the sale as boot_to_desktop

Reply:
and mainly `FIRST_BOOT_CONFIG` selection is basically a switch or in perl terms if..elsif thing

Reply:
Yes, I created it here https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18699
Nothing special, just upload some logs.